### PR TITLE
Added version to stress tests files

### PR DESCRIPTION
**Pull request by akremsa**
Part of #573 

Added version number to stress tests file names and to _Labels_ field.

Tests output file looks like this:
_stress_1_fleet_10_scale_down_1.5.0-0607041_2020-03-26_1424.json_

Labels field:
_"Labels": "Agones stress_1_fleet_10_scale_up_1.5.0-0607041"_

### CODE REVIEW

**Comment by aLekSer** [comment]
E2E test has failed:
```
time="2020-03-26 15:11:39.376" level=info msg="fleet simple-fleet-7gl2g has 0/3 ready replicas"
time="2020-03-26 15:11:39.278" level=info msg="waiting for fleet condition" fleet=simple-fleet-7gl2g
    allocator_test.go:61: deleting pods failed: pods "agones-allocator-765ccc4685-6gjjv" not found
--- FAIL: TestAllocator (1.99s)
time="2020-03-26 15:11:31.916" level=info msg="Finished cleanup."
```

File: test/e2e/framework/framework.go [hunk 1]
```diff
@@ -115,6 +116,7 @@ func NewFromFlags() (*Framework, error) {
 		"optional secret to be used for pulling the gameserver and/or Agones SDK sidecar images")
 	stressTestLevel := flag.Int("stress", 0, "enable stress test at given level 0-100")
 	perfOutputDir := flag.String("perf-output", "", "write performance statistics to the specified directory")
+	version := flag.String("version", "", "version defaults to the short hash of the latest commit")
```
Comment (aLekSer) [inline test/e2e/framework/framework.go:119]: Version is usually in form `1.5.0-89abcdef` and is defaulted in Make target not here. So it is better to say: 
```suggestion
	version := flag.String("version", "", "Agones controller version which was tested, in the form of release version plus the short hash of the latest commit")
```

Comment (akremsa) [reply test/e2e/framework/framework.go:119]: Tnanks for the suggestion

File: test/e2e/framework/framework.go [hunk 2]
```diff
@@ -115,6 +116,7 @@ func NewFromFlags() (*Framework, error) {
 		"optional secret to be used for pulling the gameserver and/or Agones SDK sidecar images")
 	stressTestLevel := flag.Int("stress", 0, "enable stress test at given level 0-100")
 	perfOutputDir := flag.String("perf-output", "", "write performance statistics to the specified directory")
+	version := flag.String("version", "", "agones controller version that was tested in the release version plus the short hash of the latest commit")
```
Comment (aLekSer) [inline test/e2e/framework/framework.go:119]:
```suggestion
	version := flag.String("version", "", "agones controller version to be tested, in the form of release version plus the short hash of the latest commit")
```

**Pull request merged**